### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by GameKn1ght** [dmdorman/hero6e-foundryvtt#257] [General]
I like to play Hero System in TTS because someone was brilliant and kind enough to develop a mod to make that possible and fun. But TTS can be demanding on players PCs, and crash and be a little frustrating. Also TTS, as a paid product, requires every player to own a copy. Still fun, but not as approachable as Foundry for players.

SO THANK YOU for the work you are putting into making this a cool mod as well...on a platform that is less demanding on PCs, and free to players! I think Mod coders are not thanked enough for their efforts. So I wanted to at least say thanks!

I am also actually kind of curious how this will play using ripper's 3D map tools as well. :)

**Comment by aeauseth** [comment]
@GameKn1ght if you get the 3D stuff working, I'd love to see some example and/or your opinion of how the works in a herosystem environment.
